guides/performance.md

```diff
@@ -65,7 +65,7 @@ class MyClass
 end
 ```
 
-This philosophy of appending to an IO instead of returning an intermediate string is present in other APIs, such as in the JSON and YAML APIs, where one needs to define `to_json(io)` and `to_yaml(io)` methods to write this data directly to an IO. You should use this strategy in your API definitions too.
+This philosophy of appending to an IO instead of returning an intermediate string provides better performance than handling intermediate strings. You should use this strategy in your API definitions too.
```

Comment:
What is the problem with referring to those examples?

Reply:
These APIs don't use IO any more, they have been changed to `to_json(JSON::Builder)` and `to_yaml(YAML::Builder)` since 0.20.4.

Reply:
Good point. Ok, can you change "provides better performance" to "yields better performance" or "results in better performance"? To "provide performance" reads a bit oddly IMO.